app/signals/apps/services/domain/signals/delete.py

```diff
@@ -0,0 +1,118 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2023 Gemeente Amsterdam
+import logging
+import uuid
+
+from django.conf import settings
+from django.core.files.storage import default_storage
+from django.db import transaction
+from django.utils import timezone
+
+from signals.apps.search.tasks import delete_from_elastic
+from signals.apps.signals.models import DeletedSignal, Signal
+from signals.apps.signals.workflow import AFGEHANDELD, GEANNULEERD, GESPLITST
+from signals.celery import app
+
+log = logging.getLogger(__name__)
+
+
+class DeleteSignalsService:
+    """
+    A service class for deleting signals in a specific state after a certain period of time.
+
+    This class provides methods for deleting signals that meet the specified criteria.
+    """
+    valid_states = [AFGEHANDELD, GEANNULEERD, GESPLITST, ]
```

Comment:
Technically this is a public property and can be overwritten or modified, should this not be a constant?
Maybe it would be good to use: https://docs.python.org/3/library/typing.html#typing.Final